modules/core/src/viewports/web-mercator-viewport.js

```diff
@@ -73,7 +76,7 @@ export default class WebMercatorViewport extends Viewport {
       width,
       height,
       pitch,
-      altitude,
+      altitude: projectionMatrix ? projectionMatrix[5] / 2 : altitude,
```

Comment:
I thought this was deprecated? Let's spell out the underlying logic here:

```js
const fov = altitudeToFov(projectionMatrix ? projectionMatrix[5] / 2 : altitude);

getProjectionParameters({fov, ...});
```

Reply:
Fixed, but requires a new math.gl release as these function were not exported